cluster/kubemark/config-default.sh

```diff
@@ -39,6 +39,20 @@ MASTER_TAG="kubemark-master"
 MASTER_IP_RANGE="${MASTER_IP_RANGE:-10.246.0.0/24}"
 CLUSTER_IP_RANGE="${CLUSTER_IP_RANGE:-10.244.0.0/16}"
 RUNTIME_CONFIG="${KUBE_RUNTIME_CONFIG:-}"
+TERMINATED_POD_GC_THRESHOLD=${TERMINATED_POD_GC_THRESHOLD:-100}
+
+# Optional: enable v1beta1 related features
+ENABLE_DEPLOYMENTS="${KUBE_ENABLE_DEPLOYMENTS:-true}"
+ENABLE_DAEMONSETS="${KUBE_ENABLE_DAEMONSETS:-true}"
+
+TEST_CLUSTER_LOG_LEVEL="${TEST_CLUSTER_LOG_LEVEL:---v=4}"
```

Comment:
I think we want the default to be the same as for scalability tests:
https://github.com/kubernetes/kubernetes/blob/master/hack/jenkins/e2e.sh#L419
which is "--v=2"